static/app/views/performance/charts/chart.tsx

```diff
@@ -176,6 +179,9 @@ function Chart({
   };
 
   if (loading) {
+    if (isLineChart) {
+      return <LineChart height={height} series={[]} {...areaChartProps} />;
+    }
```

Comment:
Can be a follow-up, but I think we typically use the `TransparentLoadingMask` over a chart to indicate loading states.

Reply:
The transparent loading mask should only appear on reload state, which I'm thinking of handling in performance widget under the hood, instead of in these charts.